torch/lib/TensorLib/HalfConvert.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
There are tlib::Half types. Just no THHalf or cuda 'half' types to avoid recreating a dependency in client code